cli/share.go

```diff
@@ -166,19 +166,20 @@ func share(c *cli.Context) {
 	b.NewLine()
 	b.Plain("Just follow these steps:").NewLine()
 	b.NewLine()
-	b.Plain("0. ").Plain(GuideTextForStepAudit(toolMode)).NewLine()
+
+	b.Plain("0. ").Plain(GuideTextForStepAudit(toolMode)).NewLine() //nolint:govet
```

Comment:
```
cli/share.go:172:23: printf: non-constant format string in call to (*github.com/bitrise-io/colorstring.Builder).Plain (govet)
        b.Plain("1. ").Plain(GuideTextForStart()).NewLine()
```

I think it's okay in this case. The colorstring builder library has no method for printing a non-format-string string